internal/cb/options.go

```diff
@@ -47,6 +48,7 @@ func DefaultCircuitBreakerOptions() *CircuitBreakerOptions {
 		MaxRetries:      0,
 		MaxFailureCount: 0,
 		ResetTimeout:    0,
+		Timeout:         time.Duration(1<<63 - 1),
```

Comment:
I keep seeing this value throughout the code base. I understand that this is max duration but can we assign it to a utility constant and use the variable instead.

Reply:
I don't want the `cb` package to depend on anything else, so intrdocude a constant in that package: https://github.com/hazelcast/hazelcast-go-client/pull/615/commits/a5f8e538d5c91e4d16758c59c2acb159f18c5a8c